test/cli/help/main/expected-stdout

```diff
@@ -32,6 +32,9 @@ Available Commands:
   local persist                  Detect new/deleted configurations and rows.
   local encrypt                  Encrypt all unencrypted secrets.
   local validate                 Validate the local directory.
+  local validate config          Validate a configuration JSON file.
+  local validate row             Validate a configuration row JSON file.
+  local validate schema          Validate a configuration/row JSON file by a JSON schema file.
```

Comment:
David sa na Slacku pytal, ci vie nejak skontrolovat, ze zmena v scheme komponenty "nerozbije" CLI.

Stav bol taky, ze bez setupovania lokalneho adresara pre projekt to neslo.

Preto som pridal tieto 3 prikazy, ktore funguju aj mimo projektoveho adresara (ale rovnako aj v) a bez tokenu.